.github/workflows/rust-cli-publish.yml

```diff
@@ -92,7 +92,7 @@ jobs:
     - name: Install Rust toolchain
       uses: dtolnay/rust-toolchain@stable
       with:
-        targets: aarch64-apple-darwin
+        targets: x86_64-apple-darwin
```

Comment:
Does this work with M-series?

Reply:
Yes, this specifies the extra target that needs to be downloaded. Previously we were building on Mac x64 runners, so aarch64 was the extra target to pull down. Now that GitHub runners are using M1 (ARM), builds started failing because x64 is the extra target to pull down 😋